lua/lvim/plugins.lua

```diff
@@ -76,6 +76,7 @@ return {
   -- Install nvim-cmp, and buffer source as a dependency
   {
     "hrsh7th/nvim-cmp",
+    event = "UIEnter",
```

Comment:
I'm a little surprised this fires for TUI since according to [the documentation](https://neovim.io/doc/user/autocmd.html#UIEnter) this is only for RPC based clients, not the default TUI.

Seems to work though :shrug: